game/core/src/me/lihq/game/GameMain.java

```diff
@@ -178,4 +187,66 @@ public void initialiseAllPeople()
             NPCs.add(npc);
         }
     }
+
+    private void initialiseClues()
+    {
+        //This is a temporary list of clues
+        List<Clue> tempClues = new ArrayList<Clue>();
+
+        tempClues.add(new Clue("Clue 1", "test Desc", new TextureRegion(Assets.CLUE_SHEET, 0, 0, 32, 32)));
+        tempClues.add(new Clue("Clue 2", "test Desc", new TextureRegion(Assets.CLUE_SHEET, 0, 0, 32, 32)));
+        tempClues.add(new Clue("Clue 3", "test Desc", new TextureRegion(Assets.CLUE_SHEET, 0, 0, 32, 32)));
+        tempClues.add(new Clue("Clue 4", "test Desc", new TextureRegion(Assets.CLUE_SHEET, 0, 0, 32, 32)));
+        tempClues.add(new Clue("Clue 5", "test Desc", new TextureRegion(Assets.CLUE_SHEET, 0, 0, 32, 32)));
+        tempClues.add(new Clue("Clue 6", "test Desc", new TextureRegion(Assets.CLUE_SHEET, 0, 0, 32, 32)));
+        tempClues.add(new Clue("Clue 7", "test Desc", new TextureRegion(Assets.CLUE_SHEET, 0, 0, 32, 32)));
+        tempClues.add(new Clue("Clue 8", "test Desc", new TextureRegion(Assets.CLUE_SHEET, 0, 0, 32, 32)));
+        tempClues.add(new Clue("Clue 9", "test Desc", new TextureRegion(Assets.CLUE_SHEET, 0, 0, 32, 32)));
+        tempClues.add(new Clue("Clue 10", "test Desc", new TextureRegion(Assets.CLUE_SHEET, 0, 0, 32, 32)));
+
+        Collections.shuffle(tempClues);
+
+        for (Room room : gameMap.getRooms())
```

Comment:
This should be broken up into lots of little functions, i.e. a method to get possible locations in each room and if needed a method to select one of them possible locations randomly. means it's easier to test and change down the line rather than a for loop in a for loop